Sim/SimG4Common/src/SingleParticleGeneratorAction.cpp

```diff
@@ -0,0 +1,92 @@
+#include "SimG4Common/SingleParticleGeneratorAction.h"
+#include <CLHEP/Random/RandFlat.h>
+
+#include "G4PhysicalConstants.hh"
+#include "G4SystemOfUnits.hh"
+#include "G4Event.hh"
+#include "G4ParticleGun.hh"
+#include "G4ParticleTable.hh"
+#include "G4ParticleDefinition.hh"
+#include "G4HEPEvtInterface.hh"
+#include "G4RunManager.hh"
+#include "G4PrimaryVertex.hh"
+#include "G4PrimaryParticle.hh"
+
+
+SingleParticleGeneratorAction::SingleParticleGeneratorAction()
+{
+	std::cout<<" SingleParticleGeneratorAction() "<<std::endl;
```

Comment:
No `cout`, please. Please use the msg-service if necessary. (Also applies to the others in this implementation.)